### PR TITLE
Smarter Textures - Fix OSX link failure

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

File: libraries/networking/src/FileCache.cpp [hunk 1]
```diff
@@ -19,13 +19,20 @@
 #include <QDir>
 
 #include <ServerPathUtils.h>
+#include <NumericalConstants.h>
```
Comment (zzmp) [inline libraries/networking/src/FileCache.cpp]: You added this but did not use it. Did you mean to put `BYTES_PER_MEGABYTES`/`BYTES_PER_GIGABYTES` inside of `NumericalConstants.h`?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8a438948b977bdca4076054708aadb3cb5f7a2cc available at: https://gist.github.com/b6602a26a12cf115e38b9493d292a0df<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9848-8a43894.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9848-8a43894.dmg

File: libraries/networking/src/FileCache.cpp (hunk 1 above)
Comment (jherico) [inline libraries/networking/src/FileCache.cpp]: initially, but I decided against fiddling with that.  I'm removing.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 43d8433d9a3c92ccc8dfb8910019cb068452fdc7 available at: https://gist.github.com/b6602a26a12cf115e38b9493d292a0df<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9848-43d8433.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9848-43d8433.dmg

**Comment by jherico** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 43d8433d9a3c92ccc8dfb8910019cb068452fdc7 available at: https://gist.github.com/b6602a26a12cf115e38b9493d292a0df<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9848-43d8433.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9848-43d8433.dmg

**Pull request merged**